### ISSUE TITLE
Mi Dual Mode Wireless Mouse ARM64 driver

### ISSUE THREAD

**Issue by AKopytenko** [IlyaSkriblovsky/hid-xiaomi#3]
The same problem with connecting the **Mi Dual Mode Wireless Mouse** to Ununtu in the **ARM64** architecture. It would be great if you made an ARM version!

**Comment by IlyaSkriblovsky** [comment]
I don't think this driver is arch-specific. Could you please check that your mouse is connected via bluetooth and check if `hid_xiaomi` module is inserted with `lsmod | grep xiaomi`?
